src/modules/addTaskToLocal.js

```diff
@@ -0,0 +1,24 @@
+class Task {
+  constructor(description, completed, id) {
+    this.description = description;
+    this.completed = completed;
+    this.id = id;
+  }
+}
+
+const addTaskToLocal = (description) => {
+  let onLocalStorage = window.localStorage.getItem('todo-tasks');
+  if (onLocalStorage === '') {
+    const task = new Task(description, false, 1);
+    window.localStorage.setItem('todo-tasks', JSON.stringify([task]));
+  } else {
+    onLocalStorage = JSON.parse(onLocalStorage);
+    const task = new Task(description, false, onLocalStorage.length + 1);
+    window.localStorage.setItem(
+      'todo-tasks',
+      JSON.stringify([...onLocalStorage, task]),
+    );
+  }
+};
```

Comment:
- [ ] On your web application, I am unable to add a task. When I added a task, nothing happened other than the page reloaded. Please make sure the user can add tasks. The tasks should appear immediately on the page and be saved in the local storage.
- After that ensure that the user can edit task descriptions and delete a task.